packages/core/src/components/loading-skeleton/_loading-skeleton.scss

```diff
@@ -0,0 +1,127 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+
+.pt-loading-skeleton-bones {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+
+@mixin bone-margin($margin) {
+  .pt-loading-skeleton-bone-#{$margin} {
+    margin-right: percentage($margin / 100);
+  }
+}
+@include bone-margin(0);
+@include bone-margin(5);
+@include bone-margin(10);
+@include bone-margin(15);
+
+.pt-loading-skeleton-bone {
+  border-radius: 2px;
+  background-color: rgba($gray1, 0.2);
+  width: 100%;
+  height: $pt-grid-size * 0.6;
+
+  &.pt-animated {
+    // negative animation delay ensures animations are synced
+    animation: 1.75s linear -7s infinite normal none running glow;
```

Comment:
`normal`, `none` and `running` are all defaults. What about `animation: ($pt-transition-duration * 20) linear ($pt-transition-duration * -70) glow;`?